### ISSUE TITLE
Enhanced interface score assessment

### ISSUE THREAD

**Issue by IlyesAbdelhamid** [FreshAirTonight/af2complex#11]
Hello,

I would like to know whether the interface score calculation between v1.2 and v.1.3 is totally different and if I can keep using the code of v.1.2.
Thank you in advance for your reply. 

Sincerely,
Ilyes

**Comment by FreshAirTonight** [comment]
Yes. The definition of interface score is the same between all versions.
